README.md

```diff
@@ -106,6 +109,20 @@ class MyTest {
 }
 ```
 
+## Configuring SASL
+
+To config the Broker to use SASL, use the annotations `@SaslMechanism` to specify the SASL
+mechanism. Use `@User` configure a user and password.  The `@User` annotation  may be repeated
+to define multiple users.
+
+```java
+class MyTest {
+@SaslMechanism("PLAIN") @User(user = "alice", password = "foo") KafkaCluster cluster;
```

Comment:
this has drifted with the refactor, need to update it to use `principals =`

Reply:
The duplication (and inconsistencies) between the javadoc and README is irksome.   We should probably just have one or the other.